mars/scheduler/mutable.py

```diff
@@ -74,12 +86,41 @@ def append_chunk_records(self, chunk_records):
     def seal(self):
         from ..worker.seal import SealActor
         self._sealed = True
+
+        # dump current buffers to worker
+        chunk_records_to_send = self._tensor._do_flush()
+        self._send_chunk_records(chunk_records_to_send)
+
+        # consolidate chunks
         for chunk in self._tensor.chunks:
             ep = self.get_scheduler(chunk.key)
             sealer_uid = SealActor.gen_uid(self._session_id, chunk.key)
             sealer_ref = self.ctx.create_actor(SealActor, uid=sealer_uid, address=ep)
-
             sealer_ref.seal_chunk(self._session_id, self._graph_key,
                                   chunk.key, self._chunk_map[chunk.key],
-                                  chunk.shape, self._record_type, self._dtype)
-        return self._graph_key, self._tensor.key, self._tensor.id, self.tensor_meta()
+                                  chunk.shape, self._record_type, self._dtype, self._fill_value)
+        # return the hex of self._graph_key since UUID is not json serializable.
+        return self._graph_key.hex, self._tensor.key, self._tensor.id, self.tensor_meta()
+
+    @log_unhandled
+    def _send_chunk_records(self, chunk_records_to_send):
+        from ..worker.dispatcher import DispatchActor
+        from ..worker.quota import MemQuotaActor
+        from ..worker.utils import put_chunk
+
+        chunk_records = []
+        for chunk_key, records in chunk_records_to_send.items():
+            record_chunk_key = tokenize(chunk_key, uuid.uuid4().hex)
+            ep = self.get_scheduler(chunk_key)
+            # register quota
+            quota_ref = self.ctx.actor_ref(MemQuotaActor.default_uid(), address=ep)
+            quota_ref.request_batch_quota({record_chunk_key: records.nbytes})
```

Comment:
ReceiverActor takes little process memory, and quota request is not needed.

Reply:
The quota is for chunk of records (the `(index, value)` record of write operations) and the record chunk may be spilled, thus the quota is required, IMO.

Reply:
ReceiverActor stores data in plasma_store or disk. This means the cost of **process memory** equals to zero when receiving data from other machines. What's more, we serialize with pyarrow with zero-copy and spill in small chunks (not chunks in Mars), hence the additional memory cost is no more than the size of these chunks. Therefore there is no need requesting for quotas before data transfer.

Reply:
Fixed. No need to requesting quota now.